apps/mull-ui/src/app/components/event-page-info/event-page-info.tsx

```diff
@@ -17,11 +17,12 @@ import { ExpandableText } from './../expandable-text/expandable-text';
 import MullButton from '../mull-button/mull-button';
 
 export interface EventPageInfoProps {
-  event: IEvent;
+  event: Partial<IEvent>;
   className?: string;
+  onButtonClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
```

Comment:
Can this be more descriptive as to what it does? `handleEventSubmission` or whatever you think is appropriate

Reply:
Sure, something like `handleMullButton` since the button will be used to create events in review mode, and join events in preview mode.